src/import/import-figma.ts

```diff
@@ -573,7 +609,7 @@ const getTextTransformValue = (t: Figma.Node<'TEXT'>): string => {
 
 // Get the style values from a text element (i.e. font family, font size, etc.)
 const getTextStyleValues = (t: Figma.Node<'TEXT'>): TextVariant => {
-  const fontFamily = t.style.fontFamily;
+  const fontFamily = `'${t.style.fontFamily}'`;
```

Comment:
I'm curious why you moved the wrapping quotes to this stage of the process rather than at the point of rendering it to a CSS property, is this something mandated by the Style Dictionary format?